sample_data/defaultOrganization.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "_id": "6637904485008f171cf29924",
+    "status": "ACTIVE",
+    "members": ["66378abd85008f171cf2990d"],
+    "admins": ["66378abd85008f171cf2990d"],
+    "groupChats": [],
+    "posts": [],
+    "pinnedPosts": [],
+    "membershipRequests": [],
+    "blockedUsers": [],
+    "name": "Hope Foundation",
+    "description": "A place where money and power doesn't matter. Free legal aid to everyone.",
+    "address": {
+      "city": "Kolkata",
+      "countryCode": "IN",
+      "dependentLocality": "Some Dependent Locality",
+      "line1": "line3",
```

Comment:
Leave as much of the address information blank as possible. The end user needs to understand that this needs to be changed.

Reply:
> Leave as much of the address information blank as possible. The end user needs to understand that this needs to be changed.

Every parameter required for `address` is necessary. So I don't think leaving it blank will be worth. But we can do one thing, can fill `default address` in every field.

Reply:
OK